sdk/src/main/java/org/openbaton/sdk/api/rest/ConfigurationAgent.java

```diff
@@ -17,37 +17,35 @@
 
 package org.openbaton.sdk.api.rest;
 
+import org.apache.http.annotation.ThreadSafe;
```

Comment:
Ah this is the wrong import. I actually wanted to use
`import net.jcip.annotations.ThreadSafe;`
which is simply an annotation for showing that this class is thread safe.
However, this would require a new dependency and I'm not sure if this is worth it.
I will remove the annotation.